active-directory-new-domain/azuredeploy.parameters.json

```diff
@@ -13,6 +13,9 @@
     },
     "dnsPrefix": {
       "value": "GEN-UNIQUE"
+    },

```

Comment:
remove this from the parameters file since this has a defaultValue in the template